frontend/js/src/user/playlists/Playlists.tsx

```diff
@@ -207,141 +230,171 @@ export default class UserPlaylists extends React.Component<
-                            this.onPlaylistCreated(p);
-                          });
-                        } else {
-                          this.onPlaylistCreated(playlist);
-                        }
-                      });
-                    }}
-                    data-toggle="modal"
-                    data-target="#ImportMusicServicePlaylistModal"
-                  >
-                    <FontAwesomeIcon icon={faItunesNote} />
-                    &nbsp;Apple Music
-                  </button>
-                </li>
-                <li>
-                  <button
-                    type="button"
-                    onClick={() => {
-                      NiceModal.show<JSPFPlaylist | JSPFPlaylist[], any>(
-                        ImportPlaylistModal
-                      ).then((playlist) => {
-                        if (Array.isArray(playlist)) {
-                          playlist.forEach((p: JSPFPlaylist) => {
-                            this.onPlaylistCreated(p);
-                          });
-                        } else {
-                          this.onPlaylistCreated(playlist);
-                        }
-                      });
-                    }}
-                    data-toggle="modal"
-                    data-target="#ImportPlaylistModal"
-                  >
-                    <FontAwesomeIcon icon={faFileImport} />
-                    &nbsp;Upload JSPF file
-                  </button>
-                </li>
-              </ul>
+                <FontAwesomeIcon icon={faUsers as IconProp} /> Collaborative
+              </Pill>
             </div>
-          )}
-        </div>
-        <div className="playlist-sort-controls">
-          <b>Sort by:</b>
-          <select
-            value={sortBy}
-            onChange={(e) => this.setSortOption(e.target.value as SortOption)}
-            className="form-control"
-            style={{ width: "200px" }}
-          >
-            <option value={SortOption.DATE_CREATED}>Date Created</option>
-            <option value={SortOption.DATE_UPDATED}>Date Updated</option>
-            <option value={SortOption.TITLE}>Title</option>
-            <option value={SortOption.CREATOR}>Creator</option>
-            <option value={SortOption.RANDOM}>Random</option>
-          </select>
+            <div className="playlist-view-controls">
+              <Pill
+                active={view === PlaylistView.GRID}
+                type="secondary"
+                onClick={() => this.setState({ view: PlaylistView.GRID })}
+                title="Grid view"
+              >
+                <FontAwesomeIcon icon={faGrid} />
+              </Pill>
+              <Pill
+                active={view === PlaylistView.LIST}
+                type="secondary"
+                onClick={() => this.setState({ view: PlaylistView.LIST })}
+                title="List view"
+              >
+                <FontAwesomeIcon icon={faStacked} />
```

Comment:
Maybe the regular list view icon here?
In any case, please add `fixedWidth`
```suggestion
                <FontAwesomeIcon icon={faList} fixedWidth/>
```